source/release-notes/5.0.txt

```diff
@@ -462,6 +462,13 @@ command to change the name of a sharded collection.
 
 .. _5.0-rel-notes-shell:
 
+``movePrimary`` Error Message For Writes During Operation
```

Comment:
Hey @ianf-mongodb  thanks for this update. We need this to be _before_  the `.. _5.0-rel-notes-shell:` reference on 463, because that reference points to the "Shell Changes" section.

Reply:
Hi Jeff, Thanks for pointing this out I have updated the PR.